colab/tflite_text_classification.ipynb

```diff
@@ -0,0 +1,514 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Copy of tflite_text_classification.ipynb",
+      "provenance": [],
+      "collapsed_sections": [
+        "dweksnWs1eZb"
+      ]
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_VsBo9D7GJ8G"
+      },
+      "source": [
+        "```\n",
+        "Copyright 2020 The IREE Authors\n",
+        "\n",
+        "Licensed under the Apache License v2.0 with LLVM Exceptions.\n",
+        "See https://llvm.org/LICENSE.txt for license information.\n",
+        "SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception\n",
+        "```"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3Mjqf_WmSkAs"
+      },
+      "source": [
+        "# TFLite text classification sample with IREE\n",
+        "\n",
+        "This notebook demonstrates how to download, compile, and run a TFLite model with IREE.  It looks at the pretrained [text classification](https://www.tensorflow.org/lite/examples/text_classification/overview) model, and shows how to run it with both TFLite and IREE.  The model predicts if a sentence's sentiment is positive or negative, and is trained on a database of IMDB movie reviews.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "W71slxNjS3SB"
+      },
+      "source": [
+        "# Setup"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "T8WMt2sPS4ft"
+      },
+      "source": [
+        "%%capture\n",
+        "!python -m pip install iree-compiler-snapshot iree-runtime-snapshot iree-tools-tflite-snapshot -f https://github.com/google/iree/releases/snapshot-20210521.291\n",
```

Comment:
let's set this to the latest release like the other notebooks
the "latest" tag suggestion on https://github.com/google/iree/issues/6046 may help keep it stable